cli/src/command/account.rs

```diff
@@ -920,81 +957,130 @@ pub async fn voting_output_command(account: &Account) -> Result<(), Error> {
-
-                if address.address().as_ref() == &required_address {
-                    if let Some(nts) = output_data.output.native_tokens() {
-                        native_tokens.add_native_tokens(nts.clone())?;
-                    }
-                    match &output_data.output {
-                        Output::Nft(nft) => nfts.push(nft.nft_id_non_null(output_id)),
-                        Output::Alias(alias) => aliases.push(alias.alias_id_non_null(output_id)),
-                        Output::Foundry(foundry) => foundries.push(foundry.id()),
-                        Output::Basic(_) | Output::Treasury(_) => {}
-                    }
-                    let unlock_conditions = output_data
-                        .output
-                        .unlock_conditions()
-                        .expect("output must have unlock conditions");
-                    let sdr_amount = unlock_conditions
-                        .storage_deposit_return()
-                        .map(|sdr| sdr.amount())
-                        .unwrap_or(0);
-
-                    amount += output_data.output.amount() - sdr_amount;
-                }
+    for (output_id, output_data) in unspent_outputs.into_iter().map(|data| (data.output_id, data)) {
+        outputs.push((output_id, output_data.output.kind_str().to_string()));
+
+        // Output might be associated with the address, but can't be unlocked by it, so we check that here.
+        // Panic: cannot fail for outputs belonging to an account.
+        let (required_address, _) = output_data
+            .output
+            .required_and_unlocked_address(current_time, &output_id, None)
+            .unwrap();
+
+        if address.inner() == &required_address {
+            if let Some(nts) = output_data.output.native_tokens() {
+                native_tokens.add_native_tokens(nts.clone())?;
+            }
+            match &output_data.output {
+                Output::Alias(alias) => aliases.push(alias.alias_id_non_null(&output_id)),
+                Output::Nft(nft) => nfts.push(nft.nft_id_non_null(&output_id)),
+                Output::Basic(_) | Output::Foundry(_) | Output::Treasury(_) => {}
             }
+            let unlock_conditions = output_data
+                .output
+                .unlock_conditions()
+                .expect("output must have unlock conditions");
+            let sdr_amount = unlock_conditions
+                .storage_deposit_return()
+                .map(|sdr| sdr.amount())
+                .unwrap_or(0);
+
+            amount += output_data.output.amount() - sdr_amount;
         }
     }
 
-    log = format!(
-        "{log}\nOutputs: {:#?}\nBase coin amount: {}\nNative Tokens: {:#?}\nNFTs: {:#?}\nAliases: {:#?}\nFoundries: {:#?}\n",
-        output_ids,
-        amount,
-        native_tokens.finish_vec()?,
-        nfts,
-        aliases,
-        foundries,
-    );
+    // base tokens table
+    formatted_table.push_str("\nBase Tokens");
```

Comment:
could use `writeln!` instead of all the newline chars

Reply:
Hmm, not sure this is the right PR for this tbh bc we do it like this in various other places, and if changed, should probably be done consistently. Also ... I don't necessarily see the benefit of just getting rid of `\n` when we also use other special chars like `\t`. Another disadvantage would be the added compilation times due to expanding the macro itself in all those places.

Reply:
I don't have an opinion on this but macro expansion is completely negligible, will not affect compilation time in any way

Reply:
I agree ... that's why it's not the main point of the argument. I think if we change it to `writeln!` then this is not the right PR for this IMHO.

Reply:
I think it would just help clean this code a bit. format is already being used so using this macro instead is not going to change anything

Reply:
well, format is not used everywhere but okay ... Thibault ... please make the call.